mycroft/skills/padatious_service.py

```diff
@@ -64,7 +64,7 @@ def train(self, message=None):
             single_thread = message.data.get('single_thread', False)
         self.finished_training_event.clear()
 
-        LOG.info('Training...')
+        LOG.info('Training...'+str(single_thread))
```

Comment:
What's this part for?

Reply:
I don't recall why I cared, to be honest.  I added it during debugging.  I can drop if you feel it is useless.